nss-day-cohort-48/fun-website-1-faith-s-children
# Initial creation of Middle TN Hike page with basic styling

Created the html file for Middle TN Hike page and give it very basic styling.

## Steps to Test

1. Pull this branch
2. Open in Chrome
3. Verify the page loads with 2 pictures and descriptive text next to them
